prbt_hardware_support/src/libmodbus_client.cpp

```diff
@@ -59,7 +59,7 @@ unsigned long LibModbusClient::getResponseTimeoutInMs()
 {
   struct timeval response_timeout;
   modbus_get_response_timeout(modbus_connection_, &response_timeout);
-  return response_timeout.tv_sec * 1000 + (response_timeout.tv_usec  / 1000);
+  return response_timeout.tv_sec * 1000l + (response_timeout.tv_usec  / 1000l);
```

Comment:
I misread it as a 1 the first time and had no idea what you were trying to do. ;-) ->
Can you use a capital L? I also think this might be more common

Reply:
Done